.scripts/fingerspelling_converter.py

```diff
@@ -0,0 +1,172 @@
+ETYMOLOGY = nested_defaultdict()
+
+
+def transform_is_two_handed(field: str, is_two_handed: dict):
+    en_field = is_two_handed.get("en") if is_two_handed else None
+    if en_field:
+        return True
+    return False
+
+
+def transform_etymology(field: str, data: dict):
+    etyms = []
+
+    languages = data.get("source_language", "").split(";")
+    signs = data.get("etymology", "").split(";")
+
+    for language, sign in zip(languages, signs):
+        newdata = {}
+        if language:
+            newdata["language"] = language.strip()
+        if sign:
+            newdata["sign"] = sign.strip()
+        etyms.append(newdata)
+    return etyms
+
+
+TRANSFORM_MAP = {
+    "id": {T: noop, D: LUKAPONA},
+    "video": {T: partial(noop, _return_if_null=dict()), D: LUKAPONA},
+    "signwriting": {T: partial(noop, _return_if_null=dict()), D: LUKAPONA},
+    "etymology": {T: transform_etymology, D: LUKAPONA},
+    "is_two_handed": {T: transform_is_two_handed, D: LUKAPONA},
+}
+
+TRANSLATABLE_MAP = {
+    "handshape": {T: noop, D: PARAMETERS},
+    "movement": {T: noop, D: PARAMETERS},
+    "placement": {T: noop, D: PARAMETERS},
+    "orientation": {T: noop, D: PARAMETERS},
+}
+
+
+def get_site(link):
+    return urllib.request.urlopen(link).read().decode("utf8")
+
+
+def clean_whitespace(s: str) -> str:
+    """Strip whitespace from beginning/end; strip duplicate whitespace other than newlines from middle"""
+    s = s.strip()
+    s = re.sub(r"\s*\n\s*", "\n", s)
+    s = re.sub(r"[ \t\r\f]+", " ", s)
+    return s
+
+
+def build_dict_from_sheet(link):
+    """stolen from jasima"""
+    raw_sheet = get_site(link)
+
+    # datasheet takes a file-like object, so we use StringIO
+    datasheet = csv.reader(StringIO(raw_sheet))
+
+    # next() consumes the first line of the sheet
+    keys = next(datasheet)
+
+    ID_COLUMN = keys.index("id")
+    keys.pop(ID_COLUMN)
+
+    data = {}
+    for row in datasheet:
+        entry = {}
+        entry_id = row.pop(ID_COLUMN)
+
+        for index, cell in enumerate(row):
+            cell = clean_whitespace(cell)
+            if not cell:
+                continue
+
+            if "/" not in keys[index]:
+                entry[keys[index]] = cell
+            else:
+                # e.g. 'def/en':
+                # outer = 'def'
+                # inner = 'en'
+                outer, inner = keys[index].split("/")
+                if outer not in entry:
+                    entry[outer] = {}
+                assert isinstance(entry[outer], dict), (
+                    "Parent key %s has non-dict child. Is the sheet malformed?" % outer
+                )
+                # if parent key has same name as a normal key
+                entry[outer][inner] = cell
+
+        data[entry_id] = entry
+
+    # Sort by id, case insensitive
+    data = {k: v for k, v in sorted(data.items(), key=lambda x: x[0].lower())}
+
+    return data
+
+
+def main():
+    os.makedirs("../fingerspelling", exist_ok=True)
```

Comment:
should we put all the files related to luka pona under a single folder?

possible arrangements:
```
luka_pona
|-- fingerspelling
|   |-- [gloss].toml
|-- signs
|   |-- [gloss].toml
```
etc. localization files would be as they are now